regexponce.go

```diff
@@ -73,6 +111,30 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+// variablesOrCallInCallExpr returns true if the provided *ast.CallExpr has
+// at least one argument and if the first argument contains a reference to a
+// variable or a function call.
```

Comment:
These comment are easy for me to understand. Thank you for explaining in comments.